spec/index.html

```diff
@@ -600,7 +600,13 @@ <h3>IRIs</h3>
         <dfn data-lt="URI" data-lt-noDefault><abbr title="Uniform Resource Identifier">URI</abbr>s</dfn>
         [[RFC3986]] that permits a wider range of Unicode characters.
         Every absolute URI and URL is an IRI, but not every IRI is an URI.
-        In RDF, IRIs are used as <em>IRI references</em>, as defined in [[RFC3987]].
+        In RDF, IRIs are used as <dfn data-lt="iri reference">IRI references</dfn>, as defined in [[RFC3987]]
+        <a data-cite="RFC3997#section-1.3">section 1.3</a>.
+        An IRI reference is common usage of an Internationalized Resource Identifier.
```

Comment:
What is this trying to say with "common usage"?

 In RDF (the data model), there are only IRIs.

"IRI references" are syntax and resolved by parsers.

Reply:
The "common usage" came out of RFC3987, and in order to make the definition in RDF Concepts, it made sense to include that language (to me, anyway). We used the term "IRI reference" later in the note, and it is also used in JSON-LD, so better to keep the definition localized, so specs don't need to reference both RDF Concepts and RFC3987.

Even though, as a concept, an "IRI reference" is syntax, and resolved before it makes it into the RDF data model, it is a commonly used term, as is "relative IRI reference" which we define later on.

Reply:
This (in RDF 1.1) 
"IRIs in the RDF abstract syntax MUST be absolute, and MAY contain a fragment identifier."

is not correct. "absolute" means something else.

Reply:
Clarified in https://github.com/w3c/rdf-concepts/pull/57/commits/d26528cf4070ebb1932c6b3d0e8946d7e1943441.